pep-0484.txt

```diff
@@ -2024,12 +2023,12 @@ A few one-off types are defined that test for single special methods
 
 Convenience definitions:
 
-* Optional, defined by ``Optional[t] == Union[t, type(None)]``
-
-* AnyStr, defined as ``TypeVar('AnyStr', str, bytes)``
+* Optional, defined by ``Optional[t] == Union[t, None]``
```

Comment:
Another good call. Perhaps the section on "Using None" should also be revised? It currently says

> When used in a type hint, the expression ``None`` is considered equivalent to ``type(None)``.

But that's false -- at least for mypy, and IIRC we once discussed this and decided that this is intentional: `type(None)` is not acceptable at all in type hints!